src/patternfly/components/Pagination/examples/pagination-bottom-example.hbs

```diff
@@ -2,10 +2,10 @@
   {{> pagination-options-menu id="pagination-options-menu-bottom-example"  options-menu--IsText="true"}}
 
   {{#> pagination-nav}}
-    {{#> button button--modifier="pf-m-plain pf-m-disabled" button--attribute='aria-label="Go to first page" aria-disabled="true"'}}
+    {{#> button button--modifier="pf-m-plain" button--attribute='disabled aria-label="Go to first page" aria-disabled="true"'}}
```

Comment:
I don't think we need `aria-disabled`, either, if the element has the `disabled` attribute. @jgiardino can you confirm?

Reply:
In this article it says: "A button with disabled as well as the button with aria-disabled will be labeled as "dimmed button" https://a11y-101.com/development/aria-disabled

Reply:
Looks like we only need "disabled"